apps/dapp-console/app/hooks/useFaucetVerifications.ts

```diff
@@ -17,7 +17,10 @@ const useFaucetVerifications = () => {
       {
         address: walletAddress,
       },
-      { enabled: !!walletAddress && !!authenticated },
+      {
+        enabled: !!walletAddress && !!authenticated,
+        refetchOnWindowFocus: false,
```

Comment:
unrelated fix in order to prevent putting the faucet into a loading state when it is not necessary